ui/react-ui/src/components/routes/users/Users.js

```diff
@@ -1,34 +1,118 @@
-import React, { Component } from 'react';
+import React, { Component, useEffect, useState } from 'react';
 import Container from 'react-bootstrap/Container';
 import Table from 'react-bootstrap/Table';
+import Modal from 'react-bootstrap/Modal';
+import Toast from 'react-bootstrap/Toast';
+import ToastHeader from 'react-bootstrap/ToastHeader'
+import ToastBody from 'react-bootstrap/ToastBody'
+import Button from 'react-bootstrap/Button';
 import BackButton from '../../bootstrapBackButton/BootstrapBackButton.js';
 
-export default class Users extends Component {
-	
-	state = {
-		isLoaded: false,
-		users: []
-	};
+export default class Users extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isLoaded: false,
+      users: [],
+      userToDelete: null,
+      toastMessage: ""
+    };
+    this.closeToast = this.closeToast.bind(this);
+    this.handleClose = this.handleClose.bind(this);
+    this.handleDelete = this.handleDelete.bind(this);
+  }
 
 	async componentDidMount() {
 		const url = 'https://nsc-fun-dev-usw2-thursday.azurewebsites.net/api/users';
 		const response = await fetch(url);
 		const usersData = await response.json();
+
 		this.setState({
 			users: usersData,
 			isLoaded: true,
 		});
 	}
 
+  showDeleteUserConfirmation(user){
+    this.setState({userToDelete: user});
+  }
+
```

Comment:
@theKunte i just pulled your branch and tested it. On my end, I was not able to delete the user. Per our conversation, you suggested to re-deploy the functions then try again. I believe @farhadbahrehmandhenry was able to test your delete methods and it worked for him. It might be something on my end. I will have to give it another try after functions are re-deployed.  Other than that, great work! 💯

Reply:
Good catch! The reason it is not working is because you need to redeploy the functions to azure and test it again. There was an issue with the delete method for HTTPTriggerUsersUserId. I fixed it in this PR but if someone merged their branch the functions will redeploy with the error. 

Just redeploy the functions test it and you will see that the delete user option works. Let me know if you encounter any issues. We can go over it later

Reply:
@theKunte just tested again after you re-deployed the functions. It works great! I was able to delete a user. Nice work!